app/controllers/admin/product_import_controller.rb

```diff
@@ -76,7 +76,7 @@ def process_data(method)
       begin
         @importer.public_send("#{method}_entries")
       rescue StandardError => e
-        render json: e.message, response: 500
+        render plain: e.message, status: :internal_server_error
```

Comment:
The UI does not render any error on the import loading screen and stalls because of the existing implementation of the `render` method. The string error message is being passed to the `json` option, which is not valid (_maybe it was valid at the time of implementation_), so the `data` was not present in the error response [here](https://github.com/openfoodfoundation/openfoodnetwork/blob/40531d769c10bba455f6300c8e6b36316516c6c6/app/assets/javascripts/admin/product_import/controllers/import_form_controller.js.coffee#L92-L95), causing the UI to not show any errors.
This is now fixed such that the string message is being returned as plain text, and its `status` option is also set respectively, so the errors on the UI sshowcorrectly now